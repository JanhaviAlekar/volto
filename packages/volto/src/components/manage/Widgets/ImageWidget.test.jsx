import React from 'react';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-intl-redux';
import { MemoryRouter } from 'react-router-dom';

import ImageWidget from './ImageWidget';

const mockStore = configureStore();

test('renders an image sizes widget component', () => {
  const store = mockStore({
    content: {
      create: {},
      data: {},
      subrequests: {
        asdasdasd: {},
      },
    },
    intl: {
      locale: 'en',
      messages: {},
    },
  });

  const component = renderer.create(
    <Provider store={store}>
      <MemoryRouter>
        <ImageWidget
          id="image"
          title="Image"
          fieldSet="default"
          onChange={() => {}}
        />
      </MemoryRouter>
    </Provider>,
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
