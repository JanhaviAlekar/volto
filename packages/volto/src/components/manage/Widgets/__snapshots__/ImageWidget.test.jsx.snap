// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an image sizes widget component 1`] = `
<div
  className="image-widget"
>
  <div
    className="inline field image field-wrapper-image"
  >
    <div
      className="ui grid"
    >
      <div
        className="stretched row"
      >
        <div
          className="four wide column"
        >
          <div
            className="wrapper"
          >
            <label
              htmlFor={null}
              id="fieldset-default-field-label-image"
            >
              Image
            </label>
          </div>
        </div>
        <div
          className="eight wide column"
        />
      </div>
    </div>
  </div>
</div>
`;
